src/js/apps/patients/worklist/worklist_app.js

```diff
@@ -270,4 +298,11 @@ export default App.extend({
       'click:edit': this.onClickBulkEdit,
     });
   },
+  setSearchState(state, searchQuery) {
+    if (searchQuery.length === 1) return;
```

Comment:
oh is this to prevent filtering unless there's a number of characters?  I think it's 3 or more would filter.  Also I think it needs to be `setState({ searchQuery ? searchQuery.length > 2 : searchQuery : '' }`. because it needs to un-search when backspacing.

Reply:
Ah, I thought it was 2 or more characters, hence the `=== 1` to handle the setting of the state when the query is empty. I'll adjust this.